deprecated-dune-v1-abstractions/ethereum/tokemak/view_tokemak_wallet_balances_daily.sql

```diff
@@ -110,6 +110,35 @@ WITH calendar AS
         --GROUP BY source, contract_address,symbol,display_name
         --order by "date" desc, symbol
         UNION
+        --CONVEX FRAX
+        SELECT "date", source,wallet_address, token_address, symbol,display_name, sum(qty) OVER (PARTITION BY wallet_address,symbol ORDER BY "date")as balance 
+        FROM (
+                SELECT "date", 5 as source,wallet_address, contract_address as token_address, symbol,display_name, sum(qty) as qty FROM (
```

Comment:
this subquery is failing. please do run it on dune & fix the errors

Reply:
Should be good now, thanks